test/pylib/scylla_cluster.py

```diff
@@ -137,6 +137,8 @@ def make_scylla_conf(mode: str, workdir: pathlib.Path, host_addr: str, seed_addr
             'truststore': 'conf/scyllacadb.pem',
         },
 
+        # Required for cql/cdc_compact_storage_test
+        'enable_create_table_with_compact_storage': True,
```

Comment:
Now I'm completely confused... If you do this, why do you need all those compact_storage fixture? How can the test test_create_table_with_compact_storage_default_config() pass with this?

By the way, this change in pylib/scylla_cluster.py will only apply to running the tests with test.py. To also affect test/cql-pytest/run, you also need the same option to test/cql-pytest/run.py.

Reply:
I think I was confused too when doing this.
I don't want this option to be enabled for all tests.
Only for the tests that require it.